ios/RNSScreenStack.mm

```diff
@@ -254,7 +278,9 @@ - (void)reactAddControllerToClosestParent:(UIViewController *)controller
 #if !TARGET_OS_TV
         _controller.interactivePopGestureRecognizer.delegate = self;
 #endif
+#ifndef RN_FABRIC_ENABLED
```

Comment:
Do you know why is it in `#ifndef RN_FABRIC_ENABLED`?

Reply:
Because this call was not present in Fabric implementation. Unfortunately I do not know why it was omitted, it looks as a perfectly valid call also for Fabric implementation. Should I remove the control directives and make it also execute on Fabric?

Reply:
Looks like it was omitted by mistake by @Ubax and I think it should be added here. It will be OK to test if it does not destroy any of the behaviors. Also would be nice to ask @kmagiera if he remembers why was it added in the first place.

Reply:
Added this call on Fabric here: [e15e071](https://github.com/software-mansion/react-native-screens/pull/1416/commits/e15e0713b75591a9608cb76c580ce467ade75267)